### PR TITLE
Clear screen after ncurses to prevent cluttering

### PR DESCRIPTION
**Pull request by asdil12**
Prevent something like https://openqa.suse.de/tests/8383000#step/rpi_wifi/27

### CODE REVIEW

File: tests/jeos/rpi_wifi.pm [hunk 1]
```diff
@@ -40,6 +40,8 @@ sub run {
     enter_cmd(get_required_var('RPI_WIFI_PSK'));
 
     assert_screen 'text-logged-in-root';
+    script_run 'clear';
```
Comment (okurz) [inline tests/jeos/rpi_wifi.pm:43]: That should be better `enter_cmd`

Comment (asdil12) [reply tests/jeos/rpi_wifi.pm:43]: why not check if it returned?

Comment (okurz) [reply tests/jeos/rpi_wifi.pm:43]: You are ignoring the result code anyway, you wouldn't loose anything

Comment (asdil12) [reply tests/jeos/rpi_wifi.pm:43]: I ignore the result code but not the fact that it returned.
This prevents a race condition as it really waits until then before the next command is typed.

**Pull request merged**